proxy/logging/LogCollationClientSM.cc

```diff
@@ -266,6 +266,9 @@ LogCollationClientSM::client_auth(int event, VIO * /* vio ATS_UNUSED */)
 
     return client_send(LOG_COLL_EVENT_SWITCH, NULL);
 
+  case VC_EVENT_INACTIVITY_TIMEOUT:
+    Debug("log-coll", "[%d]client::client_auth - Ignoring VC_EVENT_INACTIVITY_TIMEOUT", m_id);
+    return EVENT_CONT;
```

Comment:
Is there a reason that you don't do the actions of the other terminal cases (EOS and ERROR)?

Also, should you through VC_EVENT_ACTIVE_TIMEOUT in here while you are at it?